doc/api/console.md

```diff
@@ -167,6 +167,78 @@ console.assert(false, 'this message will print, but no error thrown');
 console.log('this will also print');
 ```
 
+### console.clear()
+<!-- YAML
+added: REPLACEME
+-->
+
+When `stdout` is a TTY, calling `console.clear()` will attempt to clear the
+TTY. When `stdout` is not a TTY, this method does nothing.
+
+*Note*: The specific operation of `console.clear()` can vary across operating
+systems and terminal types. For most Linux operating systems, `console.clear()`
+operates similarly to the `clear` shell command. On Windows, `console.clear()`
+will clear only the output in the current terminal viewport for the Node.js
+binary.
+
+### console.count([label = 'default'])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `label` {string} The display label for the counter. Defaults to `'default'`.
+
+Maintains an internal counter specific to `label` and outputs to `stdout` the
+number of times `console.count([label])` has been called. When the label is
+`'default'` (or `undefined`), the label is omitted from the console output.
+
+<!-- eslint-disable -->
+```js
+> console.count()
+1
```

Comment:
Technically yes, browser implementations handle this differently tho. Not sure yet which we should go with. I'd be ok with `default: 1` tho.